controllers/datadogagent/common_configmap.go

```diff
@@ -68,13 +67,24 @@ func (r *Reconciler) updateIfNeededConfigMap(dda *datadoghqv1alpha1.DatadogAgent
 	// Copy possibly changed fields
 	updateCM := oldConfigMap.DeepCopy()
 	updateCM.Data = newConfigMap.Data
+	if len(updateCM.Annotations) == 0 {
```

Comment:
It'd be clearer to check for `nil` as an empty, but initialized, map also has length == 0.

Reply:
good point - I am also updating the label a few lines below.